Dockerfile.backwards

```diff
@@ -19,7 +19,8 @@ ENV SETUPTOOLS_USE_DISTUTILS=stdlib
 RUN pip3 install --upgrade pip setuptools
 RUN pip3 install -r requirements.txt
 RUN pip3 install colcon-bundle
-
+RUN pip3 install colcon-bash
```

Comment:
why do we need this if we already have those in setup.cfg?

Reply:
Missed this. removed it.